client/src/document/ingredients/browser-compatibility-table/atoms/_bc-icons.scss

```diff
@@ -62,7 +62,7 @@
 
 .ic-deprecated {
```

Comment:
@danielhjacobs Did you see by any chance where these `ic-*` CSS classes are being used?

Reply:
I assume they're not. Perhaps they're from pre-redesign.

```
[... yari]$ grep -rl --exclude-dir=node_modules 'ic-deprecated' .
./client/src/document/ingredients/browser-compatibility-table/atoms/_bc-icons.scss
```

Reply:
Yup, it looks like it, which means we should be able to remove those `ic-*` classes from `_bc-icons.scss`.

<img width="390" alt="Screenshot 2022-03-14 at 20 44 09" src="https://user-images.githubusercontent.com/495429/158248980-9c2bf8c7-3543-4385-a687-ef14542953e9.png">

Reply:
All of them or just those three?

Reply:
At a glance, the answer is all of them:

```
[... yari]$ grep -rl 'ic-footnote' .
./client/src/document/ingredients/browser-compatibility-table/atoms/_bc-icons.scss
[... yari]$ grep -rl 'ic-altname' .
./client/src/document/ingredients/browser-compatibility-table/atoms/_bc-icons.scss
[... yari]$ grep -rl 'ic-disabled' .
./client/src/document/ingredients/browser-compatibility-table/atoms/_bc-icons.scss
[... yari]$ grep -rl 'ic-prefix' .
./node_modules/axe-core/axe.js
./node_modules/axe-core/axe.min.js
./node_modules/axe-core/lib/commons/text/is-valid-autocomplete.js
./client/src/document/ingredients/browser-compatibility-table/atoms/_bc-icons.scss
```

The other results for ic-prefix are horrific-prefix.